src/ExEss/Cms/Service/UserService.php

```diff
@@ -1,28 +1,38 @@
-<?php
-namespace ExEss\Cms\Api\V8_Custom\Service\User;
+<?php declare(strict_types=1);
 
+namespace ExEss\Cms\Service;
+
+use ExEss\Cms\Api\V8_Custom\Service\User\CommandService;
 use ExEss\Cms\Dictionary\Format;
 use ExEss\Cms\Entity\User;
+use Symfony\Component\Security\Core\Authentication\Token\TokenInterface;
 
 class UserService
 {
     private string $fallbackLocale;
 
+    private CommandService $commandService;
+
     public function __construct(
-        string $fallbackLocale
+        string $fallbackLocale,
+        CommandService $commandService
     ) {
         $this->fallbackLocale = $fallbackLocale;
+        $this->commandService = $commandService;
     }
 
-    public function getData(User $currentUser): array
+    public function getData(TokenInterface $token): array
     {
+        /** @var User $currentUser */
+        $currentUser = $token->getUser();
```

Comment:
this user might be empty, even if you have a token

Reply:
done